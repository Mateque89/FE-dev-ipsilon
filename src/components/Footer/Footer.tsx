import Box from '@mui/material/Box';
import React from 'react'
import { makeStyles } from '@mui/styles';
import { red } from '@mui/material/colors';


const useStyles = makeStyles({
    root: {
        width: "100%",
        textAlign: "center",
        position: "relative",
        bottom: 0,
        backgroundColor: red[600],
        color: "#ffff",
        fontWeight: "400",
        fontSize: "large",
        padding: "0 0 10px",
        '&>span': {
            fontWeight: '600'
        }
    }
});

function Footer() {
    const classes = useStyles();
    return (
        <Box className={classes.root}>
            <span>COVID19-stats</span>, app made for recruitment purposes by <span>Szymon Pielat</span>.
        </Box>
    )
}
export default Footer;
