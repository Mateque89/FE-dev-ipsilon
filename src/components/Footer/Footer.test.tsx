import React from 'react';
import Footer from './Footer';
import { create } from 'react-test-renderer'

it('footer renders correctly', () => {
  const tree = create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
