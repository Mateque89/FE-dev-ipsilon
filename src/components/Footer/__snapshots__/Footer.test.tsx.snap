// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders correctly 1`] = `
<div
  className="makeStyles-root-1 MuiBox-root css-0"
>
  <span>
    COVID19-stats
  </span>
  , app made for recruitment purposes by 
  <span>
    Szymon Pielat
  </span>
  .
</div>
`;
